### PR TITLE
add CV term for electron beam energy used in EAD

### PR DESCRIPTION
**Pull request by david-cox-sciex**
New CV term for electron beam energy. This is the kinetic energy of the electron beam using in dissociation methods induced by a free electron beam, such as electron-capture dissociation (ECD), electron-detachment dissociation (EDD), and electron-activated dissociation (EAD).

### CODE REVIEW

File: psi-ms.obo [hunk 1]
```diff
@@ -22503,6 +22503,15 @@ name: Stellar
 def: "Thermo Scientific Stellar mass spectrometer contains a quadrupole mass filter, a collision cell, and a quadrupole linear ion trap mass analyzer." [PSI:MS]
 is_a: MS:1000494 ! Thermo Scientific instrument model
 
+[Term]
+id: MS:1003410
+name: electron beam energy
+def: "The kinetic energy of the electron beam using in dissociation methods induced by a free electron beam, such as electron-capture dissociation (ECD), electron-detachment dissociation (EDD), and electron-activated dissociation (EAD)." [PSI:MS]
```
Comment (chambm) [inline psi-ms.obo:22509]: Typo here: "using in" -> "used in"

**Comment by mobiusklein** [comment]
The term itself looks good to me. I don't know enough mechanistically to say whether this would apply to ETD too though.

Ontologically, should we make a new parent term for all activation energies? We have `activation energy`, `collision energy`, `normalized collision energy`, and the related ramp terms as well as `supplemental collision energy`. This adds yet another term of the same concept but different mechanism.

**Review by mobiusklein — APPROVED** [review]

**Comment by david-cox-sciex** [comment]
A parent term for all activation energy terms seems good.
I considered using "activation energy" but since "electron beam energy" is about the energy the electron has, it seemed wrong to use a term that is about the final precursors activation energy. Also, when the spectrum is the result of mixed CID and EAD process we would need both "electron beam energy" and "collision energy" terms.

**Comment by chambm** [comment]
@mobiusklein Is this ready to merge or do we need to wait for the OLS stuff to settle down?

**Comment by david-cox-sciex** [comment]
I'm getting an error about it not being able to generate-owl. This branch has one new term and I incremented the data version count be one. I'm not sure why it is failing.

**Comment by mobiusklein** [comment]
We introduced a new workflow to fix an issue with Ontology Lookup Service. We will likely merge this tomorrow.

**Review by edeutsch — APPROVED** [review]

**Pull request merged**